proto/proto/penumbra/view/v1alpha1/view.proto

```diff
@@ -55,8 +55,21 @@ service ViewProtocol {
 
     // Query for the transaction hashes in the given range of blocks.
     rpc TransactionHashes(TransactionsRequest) returns (stream TransactionHashStreamResponse);
+    // Query for a given transaction hash.
+    rpc TransactionByHash(TransactionByHashRequest) returns (TransactionByHashResponse);
     // Query for the full transactions in the given range of blocks.
     rpc Transactions(TransactionsRequest) returns (stream TransactionStreamResponse);
+    // Query for the transaction perspective of the given transaction
+    rpc Perspective(PerspectiveRequest) returns (PerspectiveResponse);
+}
+
+message PerspectiveRequest {
+    bytes tx_hash = 1;
+}
+
+message PerspectiveResponse {
+    core.transaction.v1alpha1.TransactionPerspective txp = 1;
+    core.transaction.v1alpha1.Transaction tx = 2;
 }
```

Comment:
I would rename these to `TransactionPerspective{Request, Response}`, even though it's longer. I think this is more clear what they're doing.